lua/flybuf/init.lua

```diff
@@ -3,8 +3,11 @@ local nvim_buf_set_keymap = api.nvim_buf_set_keymap
 local fb = {}
 
 local function fname_path(buf)
-  local sep = vim.loop.os_uname().version:match('Windows') and '\\' or '/'
+  local sep = (not vim.opt.shellslash._value and vim.loop.os_uname().version:match('Windows') ) and '\\' or '/'
```

Comment:
```suggestion
  local sep = (not vim.opt.shellslash._value and vim.loop.os_uname().version:match('Windows')) and '\\' or '/'
```

sorry there has a space :)